atintegrators/DeltaQPass.c

```diff
@@ -111,14 +123,16 @@ ExportMode struct elem *trackFunction(const atElem *ElemData,struct elem *Elem,
         R2=atGetOptionalDoubleArray(ElemData,"R2"); check_error();
         T1=atGetOptionalDoubleArray(ElemData,"T1"); check_error();
         T2=atGetOptionalDoubleArray(ElemData,"T2"); check_error();
-        
+        chrom_maxorder=atGetOptionalLong(ElemData,"chrom_maxorder", 5); check_error();
```

Comment:
The default value should be 1, not 5. Otherwise if you give only Qpx and Qpy the pass method does a loop on chromx_arr and you can get a segmentation fault or similar error.

Reply:
done